quesma/queryparser/query_parser.go

```diff
@@ -1262,6 +1262,13 @@ func (cw *ClickhouseQueryTranslator) parseSize(queryMap QueryMap, defaultSize in
 		return defaultSize
 	} else if sizeAsFloat, ok := sizeRaw.(float64); ok {
 		return int(sizeAsFloat)
+	} else if sizeAsString, ok := sizeRaw.(string); ok {
```

Comment:
I'd suggest moving this to separate PR and merging to `main`

Reply:
By "this" I mean the whole `quesma/queryparser/query_parser.go` change of course 😉